addons/website/static/src/snippets/s_embed_code/000.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import publicWidget from "web.public.widget";
+import { _t } from "@web/core/l10n/translation";
+
+const EmbedCodeWidget = publicWidget.Widget.extend({
```

Comment:
This whole widget lacks comments, explain every step, this is not straightforward